### ISSUE TITLE
Lost modified info when going back

### ISSUE THREAD

**Issue by bLandais** [CheapTracker/CheapTracker#30]
**Scenario**:
- The user modifies an item - without push "Save" button
- He goes back with the back button
- The info are lost...

**Remediations**:
- Asking the user for confirmation ?
- Auto-save ?

**Comment by bLandais** [comment]
@ChewyBomber An opinion ? I think that it would be cool if the user is asked to "confirm" the backward move...

**Comment by ChewyBomber** [comment]
Agreed, let's do that. I personally prefer the "asking the user for confirmation" idea.
